src/components/Staking/WithdrawalsTabComparison.tsx

```diff
@@ -0,0 +1,81 @@
+import React from "react"
+import {
+  Heading,
+  Link,
+  ListItem,
+  Tab,
+  TabList,
+  TabPanel,
+  TabPanels,
+  Tabs,
+  Text,
+  UnorderedList,
+} from "@chakra-ui/react"
+import WithdrawalCredentials from "./WithdrawalCredentials"
+import ButtonLink from "../ButtonLink"
+
+interface IProps {}
+const WithdrawalsTabComparison: React.FC<IProps> = () => (
+  <Tabs>
+    <TabList>
+      <Tab>Current stakers</Tab>
+      <Tab>New stakers (not yet deposited)</Tab>
+    </TabList>
+
+    <TabPanels>
+      <TabPanel>
+        <Heading as="h3">Current stakers</Heading>
+        <UnorderedList>
+          <ListItem>
+            Some users may have provided a withdrawal address when initially
+            setting up their staking deposit—these users have nothing more they
+            need to do
+          </ListItem>
+          <ListItem>
+            The majority of stakers did not provide a withdrawal address on
+            initial deposit, and will need to update their withdrawal
+            credentials. The{" "}
+            <Link href="https://zhejiang.launchpad.ethereum.org/withdrawals">
```

Comment:
Lets use our internal Link component `src/components/Link`. This link should be external and open in a new tab.

Reply:
Oh good catch, that was my intent. Will patch

Reply:
```suggestion
            <Link href="https://zhejiang.launchpad.ethereum.org/withdrawals" isExternal>
```

If we add this it will open the link in a new tab. Some reason default functionality isn't catching this.

Reply:
Do me a favor and check your cache... this is working for me (arrow and opening in a new tab). Our `Link` component parses for this automatically so we shouldn't need this flag.